apps/andi/lib/andi/services/ingestion_delete_service.ex

```diff
@@ -12,8 +12,12 @@ defmodule Andi.Services.IngestionDelete do
   """
   @spec delete(term()) :: {:ok, SmartCity.Ingestion.t()} | {:error, any()} | {:not_found, any()}
   def delete(ingestion_id) do
+    delete(ingestion_id, :api)
+  end
+
+  def delete(ingestion_id, user_id) do
     with {:ok, ingestion} when not is_nil(ingestion) <- IngestionStore.get(ingestion_id),
-         _ <- Andi.Schemas.AuditEvents.log_audit_event(:api, ingestion_delete(), ingestion),
+         _ <- Andi.Schemas.AuditEvents.log_audit_event(user_id, ingestion_delete(), ingestion),
```

Comment:
If this is changed to user id, what happens to the logging when a delete ingestion event is sent from the api?

Reply:
Our understanding was that by calling "delete(ingestion_id)" (which is what the API does), it'd call our new function hydra head (where you left your comment).

Line 14/15 should ensure that it keeps the same behavior, might just look strange in the diff

Reply:
Just manually confirmed that this still logs as expected 👍